### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by catifish**
test pull request

### CODE REVIEW

**Comment by caticat** [comment]
lalala~~~
*Why?*

**Comment by catifish** [comment]
@caticat haha**It's a test!**

**Comment by caticat** [comment]
Ok,let it done.

File: readme.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 - fb
 - fc
 - fd
+- [ ] haha
```
Comment (caticat) [inline readme.md]: 这个有点奇怪

Comment (caticat) [inline readme.md]: 试一试review

**Pull request merged**